src/Athens-Examples/AthensSurfaceExamples.class.st

```diff
@@ -632,7 +633,7 @@ AthensSurfaceExamples class >> exampleUseForm [
 	form getCanvas fillRectangle: (20@20 corner: 50@50)  color: (Color blue).
 	
 	surf := AthensCairoSurface fromForm: form.
-	Display getCanvas drawImage: surf asForm  at: 0@0
+	surf asForm asMorph openInWindow
```

Comment:
Could it be

`surf asMorph openInWindow`?

Or do we want that in general surfaces can be transformed into forms?